client/extensions/woocommerce/app/reviews/reviews-list.js

```diff
@@ -0,0 +1,205 @@
+import { fetchReviews } from 'woocommerce/state/sites/reviews/actions';
+import {
+	areReviewsLoading,
+	areReviewsLoaded,
+	getReviews,
+	getTotalReviews
+} from 'woocommerce/state/sites/reviews/selectors';
+import { getReviewsCurrentPage, getReviewsCurrentSearch } from 'woocommerce/state/ui/reviews/selectors';
+import { getSelectedSiteWithFallback } from 'woocommerce/state/sites/selectors';
+import Pagination from 'components/pagination';
+import ReviewCard from './review-card';
+import ReviewsFilterNav from './reviews-filter-nav';
+import { updateCurrentReviewsQuery } from 'woocommerce/state/ui/reviews/actions';
+
+class ReviewsList extends Component {
+	componentDidMount() {
+		const { siteId, currentStatus } = this.props;
+		const query = {
+			page: 1,
+			search: '',
+			status: currentStatus,
+		};
+		this.props.updateCurrentReviewsQuery( this.props.siteId, { page: 1, search: '' } );
+		if ( siteId ) {
+			this.props.fetchReviews( siteId, query );
+		}
+	}
+
+	componentWillReceiveProps( newProps ) {
+		const hasAnythingChanged = (
+			newProps.currentPage !== this.props.currentPage ||
+			newProps.currentSearch !== this.props.currentSearch ||
+			newProps.currentStatus !== this.props.currentStatus ||
+			newProps.siteId !== this.props.siteId
+		);
+		if ( ! newProps.siteId || ! hasAnythingChanged ) {
+			return;
+		}
+
+		const query = {
+			page: newProps.currentPage,
+			search: newProps.currentSearch,
+			status: newProps.currentStatus,
+		};
+		if ( newProps.currentSearch !== this.props.currentSearch ) {
+			this.props.updateCurrentReviewsQuery( this.props.siteId, { page: 1, status: 'any' } );
+			query.page = 1;
+			query.status = 'any';
+		} else if ( newProps.currentStatus !== this.props.currentStatus ) {
+			this.props.updateCurrentReviewsQuery( this.props.siteId, { page: 1, search: '' } );
+			query.page = 1;
+			query.search = '';
+		}
+
+		if ( '' !== query.search ) {
+			query.status = 'any';
+		}
+
+		this.props.fetchReviews( newProps.siteId, query );
+	}
+
+	renderPlaceholders = () => {
+		return range( 5 ).map( ( i ) => {
+			return (
+				<Card key={ i } className="reviews__card">
+					<div className="reviews__placeholder">
+					</div>
+				</Card>
+			);
+		} );
+	}
+
+	renderNoContent = () => {
+		const { currentSearch, currentStatus, translate } = this.props;
+		let emptyMessage = translate( 'No pending reviews.' );
+		let lineMessage = translate( 'Your queue is clear.' );
+		if ( currentSearch ) {
+			emptyMessage = translate( 'There are no reviews matching your search.' );
+			lineMessage = '';
+		} else if ( 'approved' === currentStatus ) {
+			emptyMessage = translate( 'No approved reviews.' );
+		} else if ( 'spam' === currentStatus ) {
+			emptyMessage = translate( 'No spam reviews.' );
+		} else if ( 'trash' === currentStatus ) {
+			emptyMessage = translate( 'No deleted reviews.' );
+		} else if ( 'any' === currentStatus ) {
+			emptyMessage = translate( 'No reviews.' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **13** times:
`translate( 'Latest Reviews' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).